Homework1/Ternary/TernaryConvertor.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Text;
+
+namespace Ternary
+{
+    class TernaryConvertor
+    {
+        //Method checks for numbers satisfying the task in the range given
+        public void CheckForAppropriateInteger(int numberFrom, int numberTo)
+        {
+            for (int i = numberFrom; i <= numberTo; i++)
+            {
+                if (CheckForTwos(ConvertToTernary(i)))
+                {
+                    NumberPrint(i);
+                }
+            }
+        }
+
+        //checks how many number 2 has the string and returns true if it has two 2s
+        public bool CheckForTwos(StringBuilder ternaryString)
+        {
+            int numberOf2InString = 0;
+
+            for (int i = 0; i < ternaryString.Length; i++)
+            {
+                if (ternaryString[i] == '2')
+                {
+                    numberOf2InString++;
+                }
+            }
+            return numberOf2InString == 2;
+        }
+
+        //This method converts given number  to ternary number system and returns
+        //it as a string
+        public StringBuilder ConvertToTernary(int number)
```

Comment:
This method does not work correctly. 
The alg could be viewed at https://www.geeksforgeeks.org/ternary-number-system-or-base-3-numbers/ as an example. 

Here is the output for range 0..30.
As you can see it starts from 10, when should from 01. 
Also, it repeats ternary representations twice. 

10
11
12
10
11
12
10
11
12
100
101
102
110
111
112
120
121
122
100
101
102
110
111
112
120
121
122
1000
1001
1002
1010